frontend/packages/citizen-frontend/src/messages/MessageReadView.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+
+import styled from 'styled-components'
+import colors from '@evaka/lib-components/src/colors'
+import { defaultMargins } from '@evaka/lib-components/src/white-space'
+import { ReceivedBulletin } from '~messages/types'
+import { H3 } from '@evaka/lib-components/src/typography'
+import { messagesBreakpoint } from '~messages/const'
+import { FixedSpaceRow } from '@evaka/lib-components/src/layout/flex-helpers'
+import { formatDate } from '~util'
+
+type Props = {
+  bulletin: ReceivedBulletin
+}
+function MessageReadView({ bulletin }: Props) {
+  return (
+    <Container>
+      <Header>
+        <Title>{bulletin.title}</Title>
+        <FixedSpaceRow spacing="s">
+          <span>{bulletin.sender}</span>
+          <span>{formatDate(bulletin.sentAt)}</span>
+        </FixedSpaceRow>
+      </Header>
+      {bulletin.content.split('\n').map((text, i) => (
+        <React.Fragment key={i}>
+          <span>{text}</span>
+          <br />
+        </React.Fragment>
+      ))}
+    </Container>
+  )
+}
+
+const Container = styled.div`
+  flex-grow: 1;
+  box-sizing: border-box;
+  max-width: 100%;
+  min-height: 500px;
+  background-color: ${colors.greyscale.white};
+  padding: ${defaultMargins.m};
+
+  @media (max-width: ${messagesBreakpoint}) {
+    padding: ${defaultMargins.s};
+  }
+`
+
+const Header = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
```

Comment:
I think `flex-end` would be safer here, `baseline` works differently depending on browser.

Reply:
`flex-end` gives a wrong alignment. `center` is closer but still off

Reply:
It should be the same if you just remove the margin from `Title`.